config/default.yml

```diff
@@ -775,6 +775,14 @@ RSpec/PredicateMatcher:
   StyleGuide: https://rspec.rubystyle.guide/#predicate-matchers
   Reference: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/PredicateMatcher
 
+RSpec/PreferredSharedExampleMethods:
```

Comment:
On a different note, I suddenly recalled we have RSpec/Dialect cop 🙈😭

I sincerely apologise if that other cop does what this one does with aliases, again, it slipped my mind as I’m less involved with the project lately.

Reply:
Oh wow! I didn't know about it either. The tests for it are pretty sparse and the documentation is a bit confusing as to how to configure, but it does seem to do what I was looking to do.

```yaml
RSpec/Dialect:
  Enabled: true
  PreferredMethods:
    shared_examples_for: shared_examples
```

With this in mind, we can probably close this PR.